Cython/Build/pep517_backend/__init__.py

```diff
@@ -0,0 +1 @@
+from ._backend import *
```

Comment:
I don't think I can do that. I want to declare setuptools as a requirement in `get_requires_for_build_editable` which means I don't think I can import setuptools first.

I think this is something you'd live with in an in-tree backend (which you only use for your own project) but for something that Cython's distributing for other people to use, we don't want them to have to remember to list setuptools as a dependency too.

Reply:
Here's how you do it:
```suggestion
from contextlib import suppress

with suppress(ImportError):
    from setuptools.build_meta import *

from ._backend import *
```

I used this trick before — one subprocess wouldn't have `setuptools` but once the front-end installs it, it will be in place.

Reply:
One caveat would be that you still can't use setuptools' `get_requires_for_*()` hooks but the rest should be fine.